overlap-scripts/kubectl-kubestellar-ensure-wmw

```diff
@@ -92,25 +92,32 @@ EOF
 fi
 
 function run_kb_konnector_for_wmw() {
+    if kubectl get ns kube-system &> /dev/null; then
```

Comment:
This is just a general fix, not related to kube-bind, right ?

Reply:
It **is** related to kube-bind because the konnector needs the kube-system namespace in place otherwise it crashes.